### DISCUSSION TITLE
Welcome to Thorium-Special Discussions!

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/Thorium-Special#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by midzer** [comment]
Hi there,

I'm midzer from Munich. Found Thorium last year on woolyss and became addicted to it because I want to web to be fast for everyone:

Not only from browser performance perspective, but also for the web in general. There are too many overloaded pages around. Let's fix both worlds together :)

**Comment by gz83** [comment]
Hello everyone,

I'm gz83, the builder and tester of the Thorium browser. I discovered and started using Thorium a few months ago. Thorium's performance and some tweaks are amazing. I highly recommend Thorium to everyone.

I currently mainly compile Thorium for Windows (including regular version, AVX2 version, SSE4 version, ARM64 version, etc.), and also compile the version for Linux or Raspberry Pi.

**Comment by Alex313031** [comment]
@midzer @gz83 Thank you guys for starting the Thorium Special chat. I realized this was the only repo that didn't have discussions open, and I just used the default template for this one, instead of the custom one in the other repos. Anyone else who comes across this, feel free to post suggestions. Issues with the linux version or discussions about the main source would be better suited for the main [Thorium Repo Discussions](https://github.com/Alex313031/Thorium/discussions), but also feel free to comment here.
